### ISSUE TITLE
Ansible failure because APTITUDE not in path for Debian

### ISSUE THREAD

**Issue by AMarti96** [aws-samples/amazon-eks-custom-amis#6]
Thanks for the repo and the awesome effort behind it. I have been trying to use it to build my own AMI based on `Debian Stretch` and the build could not finish due to the following error

```
    amazon-ebs: TASK [common : be sure we have the latest version of all packages] *************
    amazon-ebs: fatal: [default]: FAILED! => {"changed": false, "msg": "Unable to find APTITUDE in path. Please make sure to have APTITUDE in path or use 'force_apt_get=True'"}
```

Steps to reproduce:
```
make AWS_REGION=eu-central-1 \
         AWS_VPC_ID=XXXX \
         AWS_SUBNET_ID=XXX \
         K8S_VERSION=1.16.12 \
         K8S_BUILD_DATE=2020-07-08 \
         CNI_PLUGIN_VERSION=v0.8.6 build-debian-stretch
```

Packer version: `1.4.4`
Ansible version `2.7.11`

Version:
Latest master [commit](https://github.com/aws-samples/amazon-eks-custom-amis/commit/d3e81edb1467cc75af4799f1cccba1ccdcaecf94)

I have also tried to build `Ubuntu 18.04` and the results have been the same.

**Comment by AMarti96** [comment]
After adding a `force_apt_get: True` in all the tasks using the module `apt` the task succeed, please update the code accordingly.

**Comment by arhea** [comment]
Thanks @AMarti96, I will get this updated!

**Comment by arhea** [comment]
We have released version v2.0-beta which fixes this issue by using `apt-get` and not `apt`.
